dcsazure_Snowflake_to_Snowflake_mask_pl/dcsazure_Snowflake_to_Snowflake_mask_pl.json

```diff
@@ -196,7 +196,7 @@
                           "type": "Expression"
                         },
                         "DF_FIELD_ALGORITHM_ASSIGNMENT": {
-                          "value": "'@{activity('Get Masking Parameters').output.runStatus.output.MaskingParameterOutput.value[0].FieldAlgorithmAssignments}'",
+                          "value": "'@{activity('Get Masking Parameters').output.runStatus.output.MaskingParameterOutput.value[0].FieldAlgorithmAssignment}'",
```

Comment:
I didn't load the template yet, but maybe you don't need this change. I think the output has an `s`

see line 715 below.
```
FieldAlgorithmAssignments = regexReplace(reduce(mapAssociation(keyValues(collect(identified_column),
```